relayer/src/chain/cosmos.rs

```diff
@@ -97,9 +98,7 @@ impl CosmosSDKChain {
         let request =
             tonic::Request::new(ibc_proto::cosmos::staking::v1beta1::QueryParamsRequest {});
 
-        let response = self
-            .block_on(client.params(request))?
-            .map_err(|e| Kind::Grpc.context(e))?;
+        let response = block_on(client.params(request)).map_err(|e| Kind::Grpc.context(e))?;
```

Comment:
I noticed the method `CosmosSDKChain::block_on` is still being used in a few spots. Should we completely remove it?

Reply:
We cannot remove it everywhere as depending on the caller stack we may end up with runtime-in-a-runtime error. Let's hope this will be cleaner when we migrate to tokio1.0